release-calendar/src/client/models/view-models.ts

```diff
@@ -15,8 +15,9 @@
  */
 
 import {Channel, Promotion} from '../../types';
+import {EventInput} from '@fullcalendar/core';
 
-export class EventInput {
+export class MyEventInput implements EventInput {
```

Comment:
Haha how about `ReleaseEventInput`?  "My" reminds me of demo or example code.

Reply:
I knew you'd come up with a better name :)